{{cookiecutter.project_slug}}/zappa_settings.yaml

```diff
@@ -5,7 +5,14 @@ production:
   app_function: "{{ cookiecutter.project_slug }}.app"
   lambda_description: "{{ cookiecutter.project_name }}"
   aws_region: "eu-west-2"
-  profile_name: "default"
+  profile_name: "devsoc"
+  vpc_config:
```

Comment:
What these settings are for?

Reply:
profile_name - new AWS account (transitioning to a new one, slowly).

vpc_config - runs lambda apps in a specific network on that account, which has a consistent outgoing IP address.